src/CodeGen_C.cpp

```diff
@@ -295,6 +296,9 @@ CodeGen_C::~CodeGen_C() {
             if (target.has_feature(Target::OpenGL)) {
                 stream << halide_internal_runtime_header_HalideRuntimeOpenGL_h << '\n';
             }
+            if (target.has_feature(Target::AMDGPUGFX900)) { // Add GFX803 later
```

Comment:
Seems the feature ought to be AMDHSA as that is the offload ABI being used. The actual architecture is provided separately. All tests using this are likely wanting to ask if this is the AMDHSA offload ABI, not which the specific target.